### PR TITLE
Improve Fuse metrics docs

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?

Improve Fuse metrics doc

### CODE REVIEW

File: docs/en/operation/Metrics-System.md [hunk 1]
```diff
@@ -121,6 +132,10 @@ scrape_configs:
       metrics_path: '/metrics/prometheus/'
       static_configs:
       - targets: [ '<WORKER_HOSTNAME>:<WORKER_WEB_PORT>' ]
+  - job_name: "alluxio standalone fuse"
+      metrics_path: '/metrics/prometheus/'
+      static_configs:
+         - targets: [ '<FUSE_WEB_HOSTNAME>:<FUSE_WEB_PORT>' ]
```
Comment (maobaolong) [inline docs/en/operation/Metrics-System.md:138]: Would you please to reduce 4 space to align the target element to the previous element `static_configs`.

**Review by maobaolong — APPROVED** [review]
Thanks for this documentation contribution, it LGTM, just left one minor comment inline.

File: docs/en/operation/Metrics-System.md (hunk 1 above)
Comment (LuQQiu) [reply docs/en/operation/Metrics-System.md]: Fixed, thanks for catching the problem, PTAL

**Comment by LuQQiu** [comment]
@maobaolong PTAL, thanks

**Review by maobaolong — APPROVED** [review]
LGTM

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**